nano/qt/qt.cpp

```diff
@@ -1919,7 +1919,7 @@ nano_qt::advanced_actions::advanced_actions (nano_qt::wallet & wallet_a) :
 	});
 
 	bootstrap->setToolTip ("Multi-connection bootstrap to random peers");
-	search_for_receivables->setToolTip ("Search for pending blocks");
+	search_for_receivables->setToolTip ("Search for ready to receive blocks");
```

Comment:
i'd suggest `"Search for ready-to-receive blocks"` here so you can't read it as "Search for ready, to receive blocks"

Reply:
Alternatively this could be `Search for ready to be received blocks` as that phrase is used in other changes in this PR.